### PR TITLE
Multithread server

### PR DESCRIPTION
**Pull request by jedi5218**

### CODE REVIEW

File: src/client/main.cc [hunk 1]
```diff
@@ -66,29 +74,51 @@ int main(int argc, char *argv[])
...
+        return false;
     }
-    printf("Processed string: %s\n", buff);
-    close(sockfd);
-    return 0;
+    printf("Server response: %s\n", buff);
+    if (std::cin.eof())
+        return false;
+    return true;
 }
```
Comment (jedi5218) [inline src/client/main.cc:124]: example comment

File: src/client/main.cc [hunk 2]
```diff
@@ -66,29 +74,51 @@ int main(int argc, char *argv[])
...
             printf("Unexpected error while sending. id #%i: %s\n", errno, strerror(errno));
         printf("Connection with server was closed before the query was sent.\n");
         close(sockfd);
-        return 0;
+        return false;
     }
     bzero(buff, sizeof(buff));
     retval = read(sockfd, buff, sizeof(buff));
+    //std::cout << "\r                                                           \r";
     if (retval <= 0)
```
Comment (serhii-vlasiuk) [inline src/client/main.cc]: why both checks?

File: src/server/packet_structs.h [hunk 3]
```diff
@@ -0,0 +1,67 @@
+//#include <pcap.h>
```
Comment (serhii-vlasiuk) [inline src/server/packet_structs.h]: remove unused includes

File: src/server/pcap_monitor.cc [hunk 4]
```diff
@@ -0,0 +1,186 @@
...
+    if (handle == NULL)
+    {
+        fprintf(stderr, "Couldn't open device %s: %s\n", dev, errbuf);
+        exit(1);
+    }
+    /* Compile and apply the filter */
+    if (pcap_compile(handle, &fp, filter_exp, 0, net) == -1)
+    {
+        fprintf(stderr, "Couldn't parse filter %s: %s\n", filter_exp, pcap_geterr(handle));
+        exit(1);
```
Comment (serhii-vlasiuk) [inline src/server/pcap_monitor.cc:47]: Is it possible to exit with close connections to all clients?

**Pull request merged**